Gems/PhysX/Code/Source/ArticulationLinkComponent.cpp

```diff
@@ -845,11 +854,96 @@ namespace PhysX
         }
         return AZ::Vector3::CreateZero();
     }
+
+    const AzPhysics::SimulatedBody* ArticulationLinkComponent::GetSimulatedBodyConst() const
+    {
+        const AZ::Entity* rootEntity = GetArticulationRootEntity();
+        const auto rootComponent = rootEntity->FindComponent<ArticulationLinkComponent>();
```

Comment:
Should either do a `nullptr` check against `rootEntity` or at least add an `AZ_Assert` here to make sure that `rootEntity` is not a `nullptr`.